lib/compat/wordpress-6.2/get-global-styles-and-settings.php

```diff
@@ -42,6 +42,64 @@ function wp_theme_has_theme_json( $clear_cache = false ) {
 	 */
 	function wp_theme_clean_theme_json_cached_data() {
 		wp_theme_has_theme_json( true );
+		wp_get_global_settings( true );
```

Comment:
Shouldn't this be calling the new method and clearing the cache?

```php
gutenberg_get_global_settings( true, null, true );
```

My guess is that's why the [typography tests are failing](https://github.com/WordPress/gutenberg/pull/45372#issuecomment-1294908211).

The typography tests do a lot of theme switching, so if it doesn't pass the if ( null === $settings_by_origin[ $origin ] ) {` check in `gutenberg_get_global_settings()` (6.2 version) it's expected to fail.